test/e2e/auth/audit.go

```diff
@@ -653,24 +656,33 @@ func expectAuditLines(f *framework.Framework, expected []auditEvent) {
 		expectations[event] = false
 	}
 
+	var err error
 	// Fetch the log stream.
-	stream, err := f.ClientSet.CoreV1().RESTClient().Get().AbsPath("/logs/kube-apiserver-audit.log").Stream()
-	framework.ExpectNoError(err, "could not read audit log")
-	defer stream.Close()
-
-	scanner := bufio.NewScanner(stream)
-	for scanner.Scan() {
-		line := scanner.Text()
-		event, err := parseAuditLine(line)
-		framework.ExpectNoError(err)
-
-		// If the event was expected, mark it as found.
-		if _, found := expectations[event]; found {
-			expectations[event] = true
+	for i := 0; i < retries; i++ {
+		stream, err := f.ClientSet.CoreV1().RESTClient().Get().AbsPath("/logs/kube-apiserver-audit.log").Stream()
+		defer stream.Close()
+		framework.ExpectNoError(err, "could not read audit log")
+
+		scanner := bufio.NewScanner(stream)
+		for scanner.Scan() {
+			line := scanner.Text()
+			event, err := parseAuditLine(line)
+
+			if err != nil {
+				continue
```

Comment:
log this error?

Reply:
I'm afraid that if something is really broken in audit log then it will just spam us with error messages (in 5k node clusters there are hundreds of thousand log lines, we don't want to see error message for each one).